opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/MessagesControllerTest.java

```diff
@@ -58,10 +59,12 @@
     private final RuntimeEnvironment env = RuntimeEnvironment.getInstance();
 
     private static class AcceptedMessageModel {
```

Comment:
why not use `AcceptedMessage` from message container?

Reply:
It may have to do with the `JerseyTest`. Anyhow, another advantage of this is that it verifies the structure of the JSON data without parsing.